### PR TITLE
CAV-653 - Fixing tests

### PR DESCRIPTION
**Pull request by mar36tin**

### CODE REVIEW

File: src/test/scala/uk/gov/hmrc/test/api/specs/VerificationSpec.scala [hunk 1]
```diff
@@ -109,18 +111,20 @@ class VerificationSpec extends BaseSpec {
...
     // a list of input
+
+    val emailRandomizer = RandomStringUtils.randomAlphabetic(1).toLowerCase
     val validEmailData = Table(
-      "a@a.com",
-      "a@a",
-      "A@A.COM",
-      "&$!@a.com",
-      "123@a.com"
+      s"$emailRandomizer@a.com",
```
Comment (johno1985) [inline src/test/scala/uk/gov/hmrc/test/api/specs/VerificationSpec.scala:117]: is there not a function within the core scala library that can do this?

**Review by johno1985 — COMMENTED** [review]
minor comment

File: src/test/scala/uk/gov/hmrc/test/api/specs/VerificationSpec.scala (hunk 1 above)
Comment (mar36tin) [reply src/test/scala/uk/gov/hmrc/test/api/specs/VerificationSpec.scala:117]: Possibly there is but was just looking at a quick fix without investing too much time into it

Comment (johno1985) [reply src/test/scala/uk/gov/hmrc/test/api/specs/VerificationSpec.scala:117]: We should randomise the email address but for now, the tests are passing fine

Comment (johno1985) [reply src/test/scala/uk/gov/hmrc/test/api/specs/VerificationSpec.scala:117]: you could just use Random.alphanumeric

File: project/Dependencies.scala [hunk 2]
```diff
@@ -9,6 +9,7 @@ object Dependencies {
     "com.typesafe.play"   %% "play-ahc-ws-standalone"  % "2.1.2"   % Test,
     "com.typesafe.play"   %% "play-ws-standalone-json" % "2.1.2"   % Test,
     "org.slf4j"            % "slf4j-simple"            % "1.7.25"  % Test,
-    "uk.gov.hmrc.mongo"   %% "hmrc-mongo-test-play-28" % "0.71.0"  % Test
+    "uk.gov.hmrc.mongo"   %% "hmrc-mongo-test-play-28" % "0.71.0"  % Test,
+    "org.apache.commons"  % "commons-lang3"            % "3.12.0"
```
Comment (johno1985) [inline project/Dependencies.scala:13]: remove this

File: src/test/scala/uk/gov/hmrc/test/api/specs/VerificationSpec.scala [hunk 3]
```diff
@@ -109,18 +113,20 @@ class VerificationSpec extends BaseSpec {
   // TODO: can probably remove this as part of CAV-426 when above tests are enabled
   Scenario("I wish to verify a valid email") {
```
Comment (johno1985) [inline src/test/scala/uk/gov/hmrc/test/api/specs/VerificationSpec.scala:114]: this scenario can now be removed

**Review by johno1985 — CHANGES_REQUESTED** [review]
minor changes needed

File: src/test/scala/uk/gov/hmrc/test/api/specs/VerificationSpec.scala [hunk 4]
```diff
@@ -106,28 +110,6 @@ class VerificationSpec extends BaseSpec {
...
-
-      Then("I should receive a notification Id")
-      verifyResponse.status shouldBe ACCEPTED
-      verifyResponse.header("Location").head shouldBe a[String]
-    }
-  }
-
   Scenario("I wish to verify an invalid email") {
     // a list of input
     val invalidEmailData = Table(
```
Comment (johno1985) [inline src/test/scala/uk/gov/hmrc/test/api/specs/VerificationSpec.scala:115]: will these need randomizing?

**Review by johno1985 — APPROVED** [review]

**Pull request merged**